### ISSUE TITLE
Perform a compatibility check between bed file specified at --targets and reference genome

### ISSUE THREAD

**Issue by GeertvanGeest** [nf-core/sarek#934]
### Description of feature

If the bed file is incompatible with the reference genome (i.e. specifies regions outside the reference), the pipeline fails only after the alignment step at base recalibration. It would be nice to have a process that compares e.g. the fasta index (or sequence dictionary) with the bed file, and throws an error if the bed file is incompatible.

**Comment by maxulysse** [comment]
I agree with you, and early fail would be better in that case

**Comment by FriederikeHanssen** [comment]
See also https://github.com/nf-core/sarek/issues/97

**Comment by FriederikeHanssen** [comment]
I think this would be a nice hackathon issue, it is fairly self-contained not too big. Do you know of any tool that can easily do this + possible other checks, like that it is sorted and so on? I briefly checked bedtools, but as far as I saw it is made for manipulating the bed file and not validating it in any form.
